### PR TITLE
Chris seo enhancement

### PR DESCRIPTION
**Pull request by eahk**
Meta data will priortise
page open graphics (OG) fields
then
page excerpt

The meta description will be using page excerpt information when og fields are null.

Screenshot 2021-07-05 at 15 30 27
Screenshot 2021-07-05 at 15 27 58

I have checked and tested the page title and page description fields but I didn't aware of the actual impacts.

### CODE REVIEW

File: templates/blocks/meta_fields.twig [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 	{% endif %}
+
 	{% if (author.name is defined and author.name is not null) %}
 		{% set meta_twitter_creator = author.name %}
 	{% elseif (post.author is defined and post.author is not null) %}
 		{% set meta_twitter_creator = post.author.name %}
 	{% endif %}
-	<meta name="description" content="{{ meta_description|raw }}">
+
+	<meta name="description" content="{{ meta_description }}">
```
Comment (hi-upchen) [inline templates/blocks/meta_fields.twig:18]: Move the `|striptags` to this line.
We should always escape the string in the last step. The reason is:

1. Someone else may need to update the string before that step. If you escape the string at the beginning, others can not change it. 
2. Escape the string at last step can make sure all the strings are escaped.

File: templates/blocks/meta_fields.twig [hunk 2]
```diff
@@ -1,19 +1,21 @@
 {% if ((post and (1 != fn('post_password_required', post))) or author is defined) %}
-	{% if (post.post_excerpt is defined and post.post_excerpt) %}
-		{% set meta_description = full_description %}
-	{% elseif (og_description is defined and og_description) %}
-		{% set meta_description = og_description %}
-	{% elseif (author.description is defined and author.description) %}
+  {% if (author.description is defined and author.description) %}
 		{% set meta_description = author.description|striptags %}
```
Comment (hi-upchen) [inline templates/blocks/meta_fields.twig:3]: Remove `|striptags` at this line.

File: templates/blocks/meta_fields.twig [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
-	{% else %}
+  {% elseif (og_description is defined and og_description) %}
+		{% set meta_description = og_description %}
+  {% elseif (post.post_excerpt is defined and post.post_excerpt) %}
+		{% set meta_description = post.post_excerpt|striptags %}
```
Comment (hi-upchen) [inline templates/blocks/meta_fields.twig:7]: Remove the `|striptags`

**Comment by eahk** [comment]
Screenshot 2021-07-05 at 16 21 59

use ```{{ og_title|e('html') }}```

**Pull request merged**